### ISSUE TITLE
Wraparound is turned off by default when calculating climatologies, leading to breaks in the climatology

### ISSUE THREAD

**Issue by s-scherrer** [TUW-GEO/pytesmo#250]
@alexgruber found some breaks when calculating climatologies with pytesmo, due to `wraparound=False` being the default in `calc_climatology`:

https://github.com/TUW-GEO/pytesmo/blob/14224155de7d7382617356e001f233470b9b58ec/src/pytesmo/time_series/anomaly.py#L96

<img width="642" alt="PastedGraphic-1" src="https://user-images.githubusercontent.com/24706421/143259992-58e251ad-f7e5-4be0-a8b2-38a52c33acb0.png">

It has been like this since the feature was introduced, 6 years ago, but should probably be set to `True` by default (I don't really see a usecase of `wraparound=False`). Can we change this, @wpreimes? @pstradio, does this affect #249?

**Comment by wpreimes** [comment]
From my side we can change the default value

**Comment by cpaulik** [comment]
pytesmo is < 1.0 so we can change the default without much issue.

I checked our code we actually use `wraparound=False` explicitly but our input is already a smoothed time series so there is less chance for a visible jump.

TLDR: Fine with me

**Comment by alexgruber** [comment]
Thanks for following up on that issue, guys.

While on it, I think I found a bug in the wraparound functionality:

It wraps around the earliest / latest doys **that contain a valid value**. This means that if, say, no valid values are available between December until February e.g., due to temperature flagging, then the moving average window considers data from both November and March that have been concatenated, which doesn't make sense....

I think the wraparound should ALWAYS wrap around the 31 Dec / 1 Jan, which could be done by adding the following line right after the "if wraparound:" statement in L190:
clim_ser = clim_ser.reindex(np.arange(366) + 1)

Do you guys agree?

**Comment by s-scherrer** [comment]
Good catch, I fully agree. We probably only have to move this line

https://github.com/TUW-GEO/pytesmo/blob/14224155de7d7382617356e001f233470b9b58ec/src/pytesmo/time_series/anomaly.py#L208

to before the `if wraparound:` line, but then it's probably best to wait for #249.

**Comment by s-scherrer** [comment]
#249 set `wraparound=True` by default, and also eliminated the second issue described here.

The changes were released with version 0.13.2 and should be on PyPI now.
